README.md

```diff
@@ -154,33 +154,32 @@ e.g.:
 #[derive(Serialize, Deserialize)]
 struct Person {
     name: String,
-    age: u8,
+    // Unsigned integer types are not supported. See the "Breaking Changes" section below.
```

Comment:
I'm not sure we need to have a comment here talking about unsigned integers

Reply:
I've removed it in 7b0bcb3.